prime-angular-frontend/src/app/shared/components/dialogs/content/send-email/send-email.component.html

```diff
@@ -0,0 +1,44 @@
+<h2 mat-dialog-title
+    class="d-flex align-items-stretch">
+  <mat-icon [color]="'primary'">help</mat-icon>
+  <span class="flex-grow-1">
+    {{ this.title }}
+  </span>
+</h2>
+
+<mat-dialog-content>
+  <span>Send email to:</span>
+
+  <div class="mt-4">
+    <button mat-stroked-button
+            class="mr-4"
+            cdkFocusInitial
+            (click)="onCancel()">Cancel</button>
+
+    <button mat-flat-button
+            color="primary"
+            class="mb-5 d-inline"
```

Comment:
Buttons are inline-block natively and we won't need margin on the bottom.  Can you remove the `class="mb-5 d-inline"` then this PR is good to go